plugins/win-capture/game-capture.c

```diff
@@ -592,21 +601,104 @@ static void load_placeholder_image(struct game_capture *gc)
 {
 	unload_placeholder_image(gc);
 
-	if (!dstr_is_empty(&gc->placeholder_img)) {
-		gs_image_file2_init(&gc->if2, gc->placeholder_img.array);
+	if (!dstr_is_empty(&gc->placeholder_image_path)) {
+		gs_image_file2_init(&gc->placeholder_image, gc->placeholder_image_path.array);
 
 		obs_enter_graphics();
-		gs_image_file2_init_texture(&gc->if2);
+		gs_image_file2_init_texture(&gc->placeholder_image);
 		obs_leave_graphics();
 	}
+	
+	TCHAR* translated_string = NULL;
+	size_t len = os_utf8_to_wcs(gc->placeholder_text.array, 0, NULL, 0);
+	if (len)
+	{
+		translated_string = malloc( (len+1)*2);
+		os_utf8_to_wcs(gc->placeholder_text.array, 0, &translated_string[0], len + 1);
+	} else {
+		return;
+	}
+
+	struct obs_video_info ovi;	
+	obs_get_video_info(&ovi);
+
+	gc->placeholder_text_height = ovi.base_height/8;
+	gc->placeholder_text_width = ovi.base_width;
+
+	BITMAPINFOHEADER bmphdr = { 0 };
+	bmphdr.biSize = sizeof(BITMAPINFOHEADER);
+	bmphdr.biWidth = gc->placeholder_text_width;
+	bmphdr.biHeight = -gc->placeholder_text_height;
+	bmphdr.biPlanes = 1;
+	bmphdr.biBitCount = 32;
+	bmphdr.biSizeImage = gc->placeholder_text_height * gc->placeholder_text_width * 4;
```

Comment:
I assume we multiple by 4 because the format is RGBA and it's 4 bytes per pixel? If so can put this 4 into a constant variable with a meaningful name?